### DISCUSSION TITLE
Pixi v8 pixel art games setup

### DISCUSSION THREAD

**Discussion by goldenratio** [pixijs/pixijs#11018] [Q&A] [answered]
Hey everyone,

How can I set `scaleMode` to "nearest" for the whole the pixi app in v8?

**Comment by bigtimebuddy** [comment, marked as answer]
```js
TextureStyle.defaultOptions.scaleMode = 'nearest';
```

**Reply by goldenratio** [reply]
thanks!

**Reply by johnoestmannmusic** [reply]
Just adding for other beginners, this line needs to be before you create your textures. It only affects textures which occur after this line.
